src/frontend/old/old_merlin.ml

```diff
@@ -87,14 +87,12 @@ let signal sg behavior =
 let rec merlin_loop input output =
   let notifications = ref [] in
   Logger.with_notifications notifications @@ fun () ->
-  let tr = Trace.null in
   match
-    Trace.enter tr "Merlin main loop"
-      ~return:(fun () -> sprintf "continue = %b")
-    @@ fun tr ->
+    (*Trace.enter tr "Merlin main loop"
+      ~return:(fun () -> sprintf "continue = %b")*)
```

Comment:
I'm not so fond of leaving dead code as comments.
Is there any reason to not simply remove this?

Reply:
Let's remove it.